src/providers/rate/rate.ts

```diff
@@ -23,7 +23,7 @@ export class RateProvider {
     this.alternatives = [];
     for (const coin of this.currencyProvider.getAvailableCoins()) {
       this.rateServiceUrl[coin] = env.ratesAPI[coin];
-      this.rates[coin] = {};
+      this.rates[coin] = { USD: 1 };
```

Comment:
PAX, USDC, and GUSD are stable coins with 1:1 ratio to USD. `https://bitpay.com/api/rates/pax` does not include USD rates or has not been implemented yet so if `USD` rates is missing, set `USD: 1`. 

We need to convince Bitpay to implement USD amount for potential erc20 tokens then we can remove this.